public/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Localhost acts as a crawl source for fake sources during integration
+# tests.  See tests/integration/fake-sources.
+
+/tests
```

Comment:
We should not put files in public/ that are not of our production deployment artifacts

Reply:
Got it, I'll move this rule up to the root gitignore.  The tests copy things to public, and then remove them.

Reply:
Ignore rule moved up to `./.gitignore`, thanks!

Reply:
Also added delete on sandbox stop, JIC.